### PR TITLE
Added backwards compability

### PR DESCRIPTION
**Pull request by andizer**

### CODE REVIEW

File: wpseo-woocommerce.php [hunk 1]
```diff
@@ -907,4 +904,59 @@ function initialize_yoast_woocommerce_seo() {
...
+class WPSEO_WooCommerce_Wrappers {
+
+	/**
+	 * Fallback for admin_header
+	 *
+	 * @param bool   $form
+	 * @param string $option_long_name
+	 * @param string $option
+	 * @param bool   $contains_files
+	 */
```
Comment (Petervw) [inline wpseo-woocommerce.php]: Missing @return phpdoc.

File: wpseo-woocommerce.php [hunk 2]
```diff
@@ -907,4 +904,59 @@ function initialize_yoast_woocommerce_seo() {
...
+		}
+
+		return self::admin_pages()->admin_header( true, 'yoast_wpseo_news_options', 'wpseo_news' );
+	}
+
+	/**
+	 * Fallback for admin_footer
+	 *
+	 * @param bool $submit
+	 * @param bool $show_sidebar
```
Comment (Petervw) [inline wpseo-woocommerce.php]: Missing @return phpdoc.

**Comment by Petervw** [comment]
CR 1 done.

**Comment by omarreiss** [comment]
CR and acceptance testing: :heavy_check_mark:

**Pull request merged**